internal/kpt/util/man/man_test.go

```diff
@@ -116,47 +116,32 @@ The \fBjava\fP package runs a container containing a java application.
 .PP
 Java server Deployment.
 
-.PP
-.RS
-
-.nf
+.EX
```

Comment:
What's going on with these changes? I don't really get these instructions. Some explanation would help :)

Reply:
Upgrade to this funky [lib](https://pkg.go.dev/github.com/cpuguy83/go-md2man/v2@v2.0.4) required changes to the expected output.

Reply:
If this is coming from kpt, was the changes on their side needed as well? Anyways, hopefully it doesn't allow some linux kernel hack.

Reply:
Still using an older version of the module.
https://github.com/kptdev/kpt/blob/main/go.mod#L9